lib/web/apiserver.go

```diff
@@ -1408,13 +1408,24 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 
 	// get tunnel address to display on cloud instances
 	tunnelPublicAddr := ""
-	if clusterFeatures.GetCloud() {
-		proxyConfig, err := h.cfg.ProxySettings.GetProxySettings(r.Context())
-		if err != nil {
-			h.log.WithError(err).Warn("Cannot retrieve ProxySettings, tunnel address won't be set in Web UI.")
-		} else {
+	assistEnabled := false // TODO(jakule)
```

Comment:
Impersonated TODO. Can I at least know what needs to be done? 😅

Reply:
sorry, i think i  misunderstood. it was related to the TODO block for below (mostly copy pasta), i thought it meant to delete the whole block and flag (didn't want you to miss it and let it always be false)

i'll just remove the comment 😅 

```go
// TODO(jakule): This part should be removed once the plugin support is added to OSS.
		if proxyConfig.AssistEnabled {
			enabled, err := h.cfg.ProxyClient.IsAssistEnabled(r.Context())
			if err != nil {
				return nil, trace.Wrap(err)
			}

			// disable if auth doesn't support assist
			assistEnabled = enabled.Enabled
		}
```

Reply:
It's fine. I'd just add `remove when plugins are implemented` as otherwise it's not clear what should be done without any context.